ProteoformSuiteInternal/ProteoformCommunity.cs

```diff
@@ -96,32 +141,145 @@ private static List<ProteoformRelation> count_nearby_relations(List<ProteoformRe
         public Dictionary<string, List<ProteoformRelation>> relate_ed()
         {
             Dictionary<string, List<ProteoformRelation>> ed_relations = new Dictionary<string, List<ProteoformRelation>>();
-            Parallel.ForEach(decoy_proteoforms, decoys =>
+            if (!Lollipop.limit_TD_BU_theoreticals)
```

Comment:
The boolean is for limiting the theoretical database to theoretical proteoforms seen in td/bu so I was just making sure to also limit the decoy database in the same way